pages/local/index.js

```diff
@@ -21,10 +22,12 @@ export async function getServerSideProps() {
   const markdownPath = join(localStaticDirectory, LOCAL_ID, "homepage.md");
   const pageMarkdown = await fs.promises.readFile(markdownPath, { encoding: "utf8"});
 
+  const props = washObject({
```

Comment:
Good think you made this `washObject` method!  He's everywhere!  The new MVP of the fontend.